include/fizzy/fizzy.h

```diff
@@ -133,6 +133,27 @@ typedef struct FizzyExternalGlobal
     FizzyGlobalType type;
 } FizzyExternalGlobal;
 
+/// External kind.
+typedef enum FizzyExternalKind
+{
+    FizzyExternalKindFunction,
+    FizzyExternalKindTable,
+    FizzyExternalKindMemory,
+    FizzyExternalKindGlobal
+} FizzyExternalKind;
+
+/// Export description.
+typedef struct FizzyExportDescription
```

Comment:
Same comment as with FizzyImportDescription: https://github.com/wasmx/fizzy/pull/683#discussion_r564742640. Please add a remark about the lifetime of `name`.

Reply:
Added.